ramls/inventory-view-instance.json

```diff
@@ -7,6 +7,10 @@
       "description": "Instance id",
       "$ref": "uuid.json"
     },
+    "isBoundWith": {
+      "description": "Records the relationship between a part of a bound-with (a holdings-record) and the bound-with as a whole (the circulatable item)",
+      "type": "boolean"
+    },
```

Comment:
This suggests a public API has been changed as part of this work.

How wrong is my understanding?